src/components/common/Actions.vue

```diff
@@ -149,6 +144,43 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
     }
     this.setIsFilingPaying(false)
   }
+
+  private get nextRoute (): string | null {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.to
+    }
+    return null
+  }
+
+  private get nextButtonLabel (): string {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.text.replace('\n', ' ')
+    }
+    return ''
+  }
+
+  private next () : any {
+    const currentStep: number| null = this.$router.currentRoute.meta
+      ? this.$router.currentRoute.meta.step : null
```

Comment:
I'm not 100% sure but this could potentially be refactoring using optional chaining? Worth a try :)  
`const currentStep: number| null = this.$router.currentRoute.meta?.step`

Reply:
Done